### DISCUSSION TITLE
Model Viewer Note Question

### DISCUSSION THREAD

**Discussion by emincanyuksel** [google/model-viewer#4554] [Q&A]
![image](https://github.com/google/model-viewer/assets/125218696/bafb0145-180a-46f8-9dc9-f4d4605894b7)

Hello everyone, can an extra text be created when the hotspot button I created here is clicked? Maybe I didn't explain this clearly. In fact, the logic is something like modal logic in bootstrap. I would be happy if you help.

**Comment by elalish** [comment]
yes, if you notice, hotspots are nothing but child HTML elements - and they can include as much element hierarchy as you want. Likewise, you can write JS that transforms them. This is all independent of MV itself - we just handle positioning them for you.
